lua/haskell-tools/hoogle/web.lua

```diff
@@ -44,11 +44,7 @@ local function setup_telescope_search()
         error("haskell-tools.hoogle-web: 'curl' executable not found! Aborting.")
         return
       end
-      opts = util.tbl_merge(opts or {}, {
-        layout_strategy = 'horizontal',
-        layout_config = { preview_width = 80 },
-        hoogle = { json = true },
-      })
+      opts = hoogle_util.merge_telescope_opts(opts, true)
```

Comment:
See my previous comment.

```suggestion
      opts = hoogle_util.merge_telescope_opts(opts, true)
      opts.hoogle = opts.hoogle or {}
      opts.hoogle.json = true,
```

Reply:
done